lib/client.js

```diff
@@ -42,6 +42,8 @@ var Client = module.exports = function(config) {
     this.brokers = new Brokers(this);
     this.topics = new Topics(this);
     this.consumers = new Consumers(this);
+
+    if (config.timeout) this.config.timeout = config.timeout;
```

Comment:
Any reason to not use the same pattern of using an override value or a default from the `DEFAULTS` object? Also, it looks like this doesn't actually work for `timeout == 0`, which is definitely an edge case and seems not too important since it is unrealistically low, but should we just be checking for `undefined` instead?